### ISSUE TITLE
Migration necessary before importing addresses

### ISSUE THREAD

**Issue by trauzti** [StefanKjartansson/django-icelandic-addresses#3]
Awesome package.

I think it should say in the README file that the user must run 

```
$ python manage.py migrate
```

before running

```
$ python manage.py import_ice_addresses
```

Also there is a new version of Stadfangaskra available here: http://www.opingogn.is/dataset/stadsetning-fasteigna/resource/bd9e812b-2e8a-4705-86a8-cd235658cc46

**Comment by StefanKjartansson** [comment]
yes, I'll change that. It's due for an upgrade anyway as the migration mechanism haves changed in django 1.7.

**Comment by trauzti** [comment]
Nice. 

I see that 

```
$ python manage.py migrate
```

is now in the README. I also see that you have run 

```
$ python manage.py makemigrations
```

and added the migrations to the project so the migration stuff from Django 1.7 is almost all there. You probably just have to remove the south migrations directory.

Perhaps the import_ice_addresses command should be added to a RunPython migration, that would conform to the Django 1.7 style. 

Once the stadfangaskra has been updated you can close this issue :8ball:
